README.md

```diff
@@ -26,6 +26,8 @@ To use make sure you have your credentials stored in a json file in the format:
 You then authorize and request by
 
 ```
+//  starting v0.2.0 use require ('@nypl/sierra-wrapper') but for versions before that (we encourage you to //  update to latest and change client to reflect the new require format) use require ('sierra-wrapper')
```

Comment:
The formatting is weird here with the second pair of `//`. That should be a new line.